gitlab-ci/SDL2-CS.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>SDL2-CS-Rolling</id>
-    <version>%version%</version>
+    <version>2000.1.1</version>
```

Comment:
Is this bit on purpose? Other than that lgtm

Reply:
This is on purpose. Since I'm running `nuget pack -Version ...` the version in the nuspec file has to be valid, or else nuget complains. So I've opted to set the version to look like what we're expecting as a form of documentation, then overriding it with the `-Version` flag.